src/hokusai/hokusai.yml

```diff
@@ -51,6 +51,7 @@ commands:
       - run:
           name: Push
           command: |
+            set +euo pipefail
```

Comment:
Have to unset bash strict mode for this to work - see https://artsy.slack.com/archives/CS3H1K4BD/p1588688718396400?thread_ts=1588688564.396300&cid=CS3H1K4BD

This should mean we can re-trigger a build and push on the same commit and CI will skip, as we expect it to, rather than hokusai refusing to push an existing tag for that git SHA1 and failing the build.

cc @jonallured @pepopowitz

Reply:
Nice one! 👍